sentinel-demo/sentinel-demo-annotation-spring-aop/src/main/java/com/alibaba/csp/sentinel/demo/annotation/aop/service/TestServiceImpl.java

```diff
@@ -16,14 +16,13 @@
 package com.alibaba.csp.sentinel.demo.annotation.aop.service;
 
 import com.alibaba.csp.sentinel.annotation.SentinelResource;
-import com.alibaba.csp.sentinel.slots.block.BlockException;
-
 import org.springframework.stereotype.Service;
 
 /**
  * @author Eric Zhao
  */
 @Service
+@SentinelResource(defaultFallback = "defaultGlobalFallback", fallbackClass = {TestServiceImpl.class})
```

Comment:
For this sample, `fallbackClass` is not required according to the resolution strategy.